web/ViewModels/PropertyViewModel.cs

```diff
@@ -114,11 +114,18 @@ public static PropertyViewModel FromDbModel(PropertyDbModel property)
                 AcceptsUnder35 = property.AcceptsUnder35,
                 AcceptsWithoutGuarantor = property.AcceptsWithoutGuarantor
             },
-            Availability = property.Availability,
-            AvailableFrom = property.AvailableFrom,
             TotalUnits = property.TotalUnits,
             OccupiedUnits = property.OccupiedUnits,
             PublicViewLink = property.PublicViewLink
         };
+
+        propertyViewModel.Availability =

```

Comment:
Similar to the comment in the service, could we make this a variable set before the model and then we can set these in the object initialisation